e2e/files/nginx-app.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
cant we make use of existing examples

Reply:
from where? Sorry I didn't understand this comment

Reply:
currently we have an example templates in `https://github.com/gluster/gcs/blob/master/deploy/examples` can we make use of it?

Reply:
got it. Let me check if I can use it. In that volume claim and pod creation is added in single yaml file.

- Also, application pods will change when we introduce more tests, we can reuse volume claims examples.
- Once we start running application tests inside pod then those examples will look complicated for users.